Makefile

```diff
@@ -13,5 +13,4 @@ run:
 	go run main.go
 
 clean:
-	find . -name build -type d | xargs rm -fr
-	find . -name vendor -type d | xargs rm -fr
+	find ./ -type d \( -name build -o -name vendor \) -prune -exec rm -fr {} 
```

Comment:
Missing `\;` or else `find` complains on macOS : `find: -exec: no terminating ";" or "+"`

The following fully works as intended: `find ./ -type d \( -name build -o -name vendor \) -prune -exec rm -fr {} \;`

Reply:
Fixed sorry !